source/docs/articles/drupal/content-delivery-network-cdn-for-file-distribution.md

```diff
@@ -21,7 +21,6 @@ If you have streaming content, large amounts of multi-national traffic or need t
 
 If you don't meet those specific criteria, strongly consider just using Pantheon's existing infrastructure. It's highly optimized and can handle terrifying amounts of traffic. Many Enterprise clients with very high-profile sites do not use CDNs and rely on Pantheon's optimized stack to deliver the performance their customers demand.
 
-<div class="alert alert-info" role="alert">
```

Comment:
It was broken and I couldn't figure out what was going on with it, so I removed it (see it on the docs site): https://pantheon.io/docs/articles/drupal/content-delivery-network-cdn-for-file-distribution/
I'd prefer not to add background color to definitions; it's likely to be ignored because it looks like a tip or unessential info.